tests/Stecman/Component/Symfony/Console/BashCompletion/HookFactoryTest.php

```diff
@@ -91,9 +92,7 @@ protected function assertSyntaxIsValid($code, $syntaxCheckCommand, $context)
 
             $status = proc_close($process);
 
-            if ($status !== 0) {
-                $this->fail("Syntax check for $context failed:\n$output");
-            }
+            $this->assertEquals(0, $status, "Syntax check for $context failed:\n$output");
```

Comment:
That's not equivalent construct, because `assertEquals` doesn't do `===` internally. Use `assertSame` instead.

Reply:
Done